jwst/master_background/master_background_step.py

```diff
@@ -73,7 +77,25 @@ def process(self, input):
                 self.record_step_status(result, 'master_background', success=False)
 
                 return result
+            # Check if subtract_background is set to False -> skip step
+            if self.subtract_background is not None and not self.subtract_background:
+                self.log.warning(
+                    "Not subtracting masterbackground, subtract_background set to False")
+                result = input_data.copy()
+                self.record_step_status(result, 'master_background', success=False)
+                return result
+            # Check if subtract_background is None but the background was
+            # subtracted in calspec2 background step  -> skip step
+            if self.subtract_background is None and \
+                input_data.meta.cal_step.back_sub == 'COMPLETE':
```

Comment:
This will fail for a `ModelContainer`, as it has no `meta.cal_step.back_sub` attribute.  It's member models do.

This suggests that a `get_step_status()` method might be useful.

Reply:
A quick-n-dirty approach would be to test for `ModelContainer` and if it is, then only check the value in the first model. If it isn't `ModelContainer`, then use the existing test above.

Reply:
Btw, this is why the unit tests below are failing.  I like Howards quick-n-dirty approach, but as we discussed this morning, it would be good to make sure all models pass the above check, not just the first one.